common/scripted_triggers/00_decisions.txt

```diff
@@ -73,7 +74,6 @@ is_tier_2_formable_trigger = {
 		tag = NUM #Numidia
 		tag = PRG #Phrygia Formable
 		tag = PTU #Pontus Formable
-		tag = PON #Pontus

```

Comment:
there may be some unintended effect of removing this. there's a vanilla decision to reform pontus if it disappears, which would then allow the formation of PTU at tier 2 for free.